module/network/RobotCommunication/src/RobotCommunication.cpp

```diff
@@ -0,0 +1,53 @@
+#include "RobotCommunication.hpp"
+#include "extension/Configuration.hpp"
+
+#include "message/input/GameState"
+#include "message/localisation/FilteredBall.hpp"
+
+namespace module::network {
+
+    using extension::Configuration;
+
+    RobotCommunication::RobotCommunication(std::unique_ptr<NUClear::Environment> environment)
+        : Reactor(std::move(environment)) {
+
+        on<Configuration>("RobotCommunication.yaml").then([this](const Configuration& config) {
+            // Use configuration here from file RobotCommunication.yaml
+            this->log_level = config["log_level"].as<NUClear::LogLevel>();
+            port            = config["port"].as<uint>();
+        });
+
+        on<Trigger<FilteredBall>,
+           With<Ball>,
+           With<Stability>,
+           With<WalkCommand>,
+           With<Kick>,
+           With<Field>,
+           With<GameStateProto>,
+           With<GlobalConfig>>()
+            .then([this](const FilteredBall& ball,
+                         const Ball& ballData const Kick& kick,
+                         const WalkCommand& walk,
+                         const Field& field,
+                         const GameStateProto& gameState const GlobalConfig& config) {
```

Comment:
```suggestion
                         const GameStateProto& game_state const GlobalConfig& config) {
```
we use snake_case for variables 
see https://nubook.nubots.net/guides/general/code-conventions